stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsControllerImpl.kt

```diff
@@ -115,7 +111,7 @@ internal class QueryChannelsControllerImpl(
             // - suspend/wait for a few seconds (yuck, lets not do that)
             // - post the refresh on a livedata object with only channel ids, and transform that into channels (this ensures it will get called after postValue completes)
             // - run the refresh channel call below on the UI thread instead of IO thread
-            scope.launch(Dispatchers.Main) {
+            domainImpl.scope.launch(Dispatchers.Main) {
```

Comment:
What if we provide scope as an argument in the constructor? It helps us write tests

Reply:
We can refactor this later if we need easier mocking for the scope in tests. For now, tests work fine through mocking the domain's scope.